packages/contracts/hardhat.config.ts

```diff
@@ -93,10 +93,10 @@ const config: HardhatUserConfig = {
       sepolia: process.env.ETHERSCAN_KEY || '',
       polygon: process.env.POLYGONSCAN_KEY || '',
       polygonMumbai: process.env.POLYGONSCAN_KEY || '',
-      arbitrumOne: process.env.ARBISCAN_KEY || '',
-      arbitrumTestnet: process.env.ARBISCAN_KEY || '',
       baseMainnet: process.env.BASESCAN_KEY || '',
       baseGoerli: process.env.BASESCAN_KEY || '',
+      arbitrumOne: process.env.ARBISCAN_KEY || '',
+      arbitrumGoerli: process.env.ARBISCAN_KEY || '',
```

Comment:
I double-checked that these are correct: https://github.com/NomicFoundation/hardhat/blob/f0ca6c8ff7ec48031ffaf368212b8dbe5bf5587e/packages/hardhat-verify/src/internal/chain-config.ts#L174